src/tic_tac_toe/input.clj

```diff
@@ -5,13 +5,20 @@
 
 (defn prompt-terminal [question]
```

Comment:
If I press `enter` when being prompted for input, the application crashes from a RuntimeException. Is this really an exception? I feel like I should be able to input incorrect values and just be prompted that my input is invalid.

https://clojuredocs.org/clojure.core/try